persistence/src/test/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.flyway.baseline-on-migrate=true
+spring.flyway.locations=classpath:db/test
```

Comment:
Warum benutzt du hier nicht die normalen Migrationsskripte?

Reply:
weil die nicht laufen - wie bereits erklärt.

Reply:
Okey, alles klar. Dürfen wir nur nicht vergessen, beide Skripte immer anzupassen. Die H2 läuft in MySQL Modus?

Reply:
das ist mir klar- geht momentan nicht anders